src/Jobs/Middleware/RecordTransaction.php

```diff
@@ -9,10 +9,7 @@ class RecordTransaction
     /**
      * Wrap the job processing in an APM transaction.
      *
-     * @param mixed    $job
      * @param callable $next
-     *
-     * @return mixed
      */
     public function handle($job, $next)
```

Comment:
`mixed` type was made available in PHP8.0 which is the min supported version, so we could add it as a type here along with callable and get rid of phpdocs for types.